krabcake/rs_hello/src/lib.rs

```diff
@@ -578,19 +603,19 @@ unsafe fn check_use_1(addr: vg_addr, shadow_addr: vg_ulong) {
     });
     match lookup {
         None => {
-            alert!(b"ALERT no stack for address 0x%08llx even though we are accessing it via pointer with tag %d\n\0",
+            alert!(b"ALERT no stack for address 0x%08llx even though we are accessing it via pointer with tag %s\n\0",
                            STACKS.get_stack_dbg_id_or_assign(addr),
-                           shadow_addr);
+                           tag_s.as_ptr());
         }
         Some((stack_dbg_id, false, _, _)) => {
-            alert!(b"ALERT could not find tag in stack for address 0x%08llx even though we are accessing it via pointer with tag %d\n\0",
+            alert!(b"ALERT could not find tag in stack for address 0x%08llx even though we are accessing it via pointer with tag %s\n\0",
```

Comment:
Based on VgPlainUmsgWriter from https://github.com/pnkfelix/krabcake-vg/pull/27/commits/834564206df8b9cb4cb30f9bee065aa0b1e19d88 , we can probably replace all this C formatting string stuff with Rust based formatting.

But you don't need to do that in this PR. Its just a clean-up task for the future.